src/APICore.ts

```diff
@@ -211,11 +214,10 @@ export default class API {
     private buildRequestInit(
         method: string,
         args: CoveoPlatformClientRequestInit | undefined,
-        prefilled?: RequestInit
+        prefilled?: RequestInit,
     ): RequestInit {
-        const {responseBodyFormat, ...requestArgs} = args ?? {};
+        const {responseBodyFormat: _, ...requestArgs} = args ?? {};
```

Comment:
Only manual change I did to shush a 'unused var' warning: Complete the object destructuring with a reassign to `_` to mark it as 'unused'